Packages/macOS/CmuxControlSocket/Sources/CmuxControlSocketAtomicsC/CmuxControlSocketAtomicsC.c

```
@@ -0,0 +1,46 @@
+#include "CmuxControlSocketAtomicsC.h"
+
+void CmuxControlSocketAtomicPointerInitialize(
+    CmuxControlSocketAtomicPointerStorage *storage,
+    uintptr_t initialValue
+) {
+    atomic_init(&storage->value, initialValue);
+}
+
+uintptr_t CmuxControlSocketAtomicPointerLoadAcquire(
```

Cppcheck (2.21.0)

[style] The function 'CmuxControlSocketAtomicPointerLoadAcquire' is never used.

(unusedFunction)